packages/apps/tools/src/components/Global/ChainSelect/index.tsx

```diff
@@ -36,12 +33,10 @@ const ChainSelect: FC<ChainSelectProps> = ({
   return (
     <Select
       {...rest}
-      label="Chain ID"
-      id={ELEMENT_ID}
```

Comment:
Out of curiosity; why remove the `id` prop?

Reply:
it is redundant